Issue installing thash/asana

Hello,
When trying to install from homebrew I'm getting this error:

go
get
github.com/thash/asana

# github.com/thash/asana/commands
src/github.com/thash/asana/commands/due.go:19:54: invalid operation: c.Args()[1] (type cli.Args does not support indexing)

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/thash/homebrew-asana/issues

go version go1.15.2 darwin/amd64
MacOS 10.15.6

--- a/asana.rb
+++ b/asana.rb
@@ -10,7 +10,7 @@ class Asana < Formula
 
   def install
     ENV['GOPATH'] = buildpath
-    system 'go', 'get', 'github.com/codegangsta/cli'
+    system 'go', 'get', 'github.com/urfave/cli'
     system 'go', 'get', 'github.com/thash/asana'
     system 'go', 'build', '-o', 'asana'
     bin.install 'asana'

